store/file/header.go

```diff
@@ -79,6 +79,20 @@ func (h *headerV0) Size() int {
 	return headerVOSize + 1
 }
 
+func (h *headerV0) RootsSize() int {
+	// axis roots are stored in two parts: row roots and column roots, each part has size equal to
+	// the square size. Thus, the total amount of roots is equal to the square size * 2.
+	return share.AxisRootSize * h.SquareSize() * 2
+}
```

Comment:
OPtinal: RootsSize and OffsetWithRoots is used only for ods file. Can be hidden away from whole header surface, if implemented as methods on `ods` file